data/com.canonical.Unity.gschema.xml

```diff
@@ -160,7 +160,8 @@
         'application:///dialer-app.desktop',
         'application:///messaging-app.desktop',
         'application:///address-book-app.desktop',
-        'application:///ubuntu-system-settings.desktop', 
+        'application:///ubuntu-system-settings.desktop',
+        'application:///openstore.openstore-team/current-user-version',
```

Comment:
Shouldn't this be `appid://openstore.openstore-team/current-user-version` i think the `application://` url only supports a desktop file and not the current-user-version syntax